test/fixtures.yml

```diff
@@ -1286,6 +1286,9 @@ shift4:
   client_guid: YOUR_CLIENT_ID
   auth_token: YOUR_AUTH_TOKEN
 
+shift4_v2:
+  secret_key: pr_test_tXHm9qV9qV9bjIRHcQr9PLPa
```

Comment:
Is this a key that can be public to everyone?

Reply:
Yes, this is the public one in shift4 API documentation!